qt/scientific_interfaces/Indirect/ISISCalibration.cpp

```diff
@@ -42,6 +42,8 @@ namespace CustomInterfaces {
 ISISCalibration::ISISCalibration(IndirectDataReduction *idrUI, QWidget *parent)
     : IndirectDataReductionTab(idrUI, parent), m_lastCalPlotFilename("") {
   m_uiForm.setupUi(parent);
+  setOutputPlotOptionsPresenter(std::make_unique<IndirectPlotOptionsPresenter>(
+      std::move(m_uiForm.ipoPlotOptions), this, PlotWidget::SpectraBin));
```

Comment:
I don't think we want to transfer ownership of this to the presenter so this should probably be passed as a raw pointer (similar to uses of `m_uiForm->fitDataView`). Similarly for other uses.